components/treelist/virtual-scrolling.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Virtual Scrolling
+page_title: TreeList - Sorting
+description: Enable and configure the virtual scrolling of rows in the TreeList for Blazor.
+slug: treelist-virtual-scrolling
+tags: telerik,blazor,treelist,virtual,rows,scrolling
+published: True
+position: 25
+---
+
+# TreeList Virtual Scrolling
+
+
+Virtual scrolling provides an alternative to paging. Instead of utilizing a pager, the user vertically scrolls through all records in the data source.
+
+To enhance rendering performance, the TreeList reuses the same set of HTML elements. As the next data loads, a loading indicator appears on the cells. If the user scrolls back up after scrolling down to the next set of rows, the previous data reloads from the data source, similar to regular paging, with the scroll distance determining the data to be loaded.
+
+## Enabling Virtual Scrolling
+
+* Set the `ScrollMode` parameter to `TreeListScrollMode.Virtual` (the default value is `Scrollable`).
+* Set the `Height` and `RowHeight` parameters.
```

Comment:
You can mention explicitly here that one parameter is string, while the other is decimal.